recipes/xitorch/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "xitorch" %}
+{% set version = "0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/xitorch/xitorch/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
There is a PyPI release, could that also be used? Just to save CI.

Reply:
The PyPI sdist is broken: https://github.com/xitorch/xitorch/issues/16